guidelines/index.html

```diff
@@ -596,6 +596,14 @@
 					<p>When an authenticated session expires, the user can continue the activity without loss of data after re-authenticating.</p>
 				</section>
 			</section>
+      
+      <section class="sc">
+					<h4>Animation from interactions</h4>
+					<p class="conformance-level">A</p>
+					<p class="change">Unchanged</p>
+					<p>For significant animation triggered by a user action that is not an essential part of the action, there is a mechanism for the user to pause, stop or hide the animation whilst still performing the same action.</p>
```

Comment:
I believe so, but the definition itself (below) needs to be bolstered. Idea is to weed out "irrelevant" / non-problematic animations (e.g. something that's very subtle, like a very gentle colour change or similar, which nominally would be "animation" but doesn't cause problems to users)